controller/src/main/java/io/pravega/controller/store/stream/tables/State.java

```diff
@@ -16,5 +16,8 @@
     UNKNOWN,
     CREATING,
     ACTIVE,
+    UPDATING,
```

Comment:
We are adding states to avoid conflicting updates to metadata which were originally protected via locking. 
Updating is set when UpdateConfiguration on a stream is performed. 
We will take up moving update/seal etc from task framework to lockless operations. But for now, state acts as a barrier to ensure multiple updates to metadata are not performed concurrently.